.github/workflows/main.yml

```diff
@@ -44,7 +44,7 @@ jobs:
 
       - name: Install Embedded Arm Toolchain arm-none-eabi-gcc
         if:   steps.cache-toolchain.outputs.cache-hit != 'true'  # Install toolchain if not found in cache
-        uses: fiam/arm-none-eabi-gcc@v1.0.2
+        uses: fiam/arm-none-eabi-gcc@v1.0.4
```

Comment:
This fixes this error in the build:
![image](https://user-images.githubusercontent.com/5289520/127422287-49cfef84-172b-4269-98e5-f138c1821975.png)

See https://github.com/fiam/arm-none-eabi-gcc/issues/9